pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -129,7 +129,7 @@ if [ "$NIX_@infixSalt@_DONT_SET_RPATH" != 1 ]; then
             # copied to $out/lib.  If not, we're screwed.
             continue
         fi
-        for path in "$dir"/lib*.so; do
+        for path in "$dir"/*; do
```

Comment:
`*.so*` would work, but I do not want to remember or force others to keep in mind that this wildcard should be wider than anything that goes in `libs`.